src/panels/PanelBase.js

```diff
@@ -97,41 +97,45 @@ export default {
   },
   methods: {
     toggle() {
-      this.localExpanded = !this.localExpanded;
-      if (this.localExpanded) {
-        this.isCached = true;
-      }
-    },
-    expand() {
-      this.localExpanded = !this.localExpanded;
-      if (this.localExpanded) {
+      if (!this.isCached && !this.localExpanded) {
+        // Let the content load before toggling localExpanded which triggers the animation,
+        // as we need to know the scrollHeight of the content for the animation to work.
         this.isCached = true;
+        this.$nextTick(() => this.localExpanded = true);
```

Comment:
Not necessarily, but I've updated the comment to be less confusing. `content` -> `retriever`

This is how it would play if its not fetched by the next tick 😁 
![panelanim](https://user-images.githubusercontent.com/3306138/77049674-78104600-6a03-11ea-99d1-bf6e8ffe0370.gif)

Reply:
Ah okay, I didn't realise that we wanted to show the `Loading...` text 😛 

This looks fine to me! Maybe we can add a spinner or something to make it prettier, but that's a (low priority) problem for next time.